src/Symfony/Component/Messenger/Transport/Sender/SendersLocator.php

```diff
@@ -46,9 +46,9 @@ public function getSenders(Envelope $envelope, ?bool &$handle = false): iterable
         $seen = array();
 
         foreach (HandlersLocator::listTypes($envelope) as $type) {
-            foreach ($this->senders[$type] ?? array() as $sender) {
+            foreach ($this->senders[$type] ?? array() as $alias => $sender) {
                 if (!\in_array($sender, $seen, true)) {
-                    yield $seen[] = $sender;
+                    yield $alias => $seen[] = $sender;
```

Comment:
Called $name above. Should $alias everywhere for consistency?

Reply:
Whoops. I'v missed the ones above, thanks.